earth_enterprise/src/fusion/autoingest/sysman/gefusion

```diff
@@ -105,6 +129,22 @@ stop() {
   fi
 }
 
+status() {
+  if check_if_master ; then
+	check_daemon gesystemmanager
+    gesystemmanager_returncode=$?
+  else
+    gesystemmanager_returncode=0
+  fi
+  check_daemon geresourceprovider
+  geresourceprovider_returncode=$?
+  if [ $gesystemmanager_returncode -eq 0 ] && [ $geresourceprovider_returncode -eq 0 ] ; then
```

Comment:
Minor: Technically, the next 5 lines are equivalent to the following, if you want to be shorter:

```
[ "$gesystemmanager_returncode" = 0 ] && [ "$geresourceprovider_returncode" = 0 ]
```

Also, I think something like the following may look shorter and use less variable names:

```BASH
return_code = 0
# . . .
check_daemon gesystemmanager || return_code=1
# . . .
check_daemon geresourceprovider || return_code=1

return "$return_code"
```

Reply:
Much nicer, thanks!